### PR TITLE
Task update

### PR DESCRIPTION
**Pull request by ElizabethlawalCYF**
Task:
Talk through your fears, get feedback, and work together to  find solutions to some of the challenges you face. 

After 20 Minutes, it’s time to swap. 

If you’re feeling brave, share your challenges to the group and ask for feedback. 

10 minutes

Task for class: 
Each person has to give one piece of advice to support their fellow team members.

## What does this change?

Module: Iteration 
Week(s): 2 

## Description

Added update to PD task 

## Who needs to know about this?

@SallyMcGrath 

## Rendered Pages

-----
[View rendered docs/fundamentals/week-2/session.md](https://github.com/CodeYourFuture/syllabus/blob/ElizabethlawalCYF-patch-1/docs/fundamentals/week-2/session.md)

### CODE REVIEW

File: docs/fundamentals/week-2/session.md [hunk 1]
```diff
@@ -145,7 +145,41 @@ Read ahead to Ship It to see where to paste your code.
...
+When it comes to showing your work in progress, what are your three biggest fears?
+When it comes to showing your work in progress, how will you overcome your three biggest fears? 
+When it comes to showing your work in progress, what’s the absolute worst thing that can happen? 
+When it comes to showing your work in progress, what is the best thing that can happen? 
+When it comes to the working world, how will you support your team mates?
+When it comes to the working world, what do you need from your team mates? 
+What’s a better outcome, making a mistake and learning something new or never making mistakes and never learning at all? 
+
+**Task:**
+Talk through your fears, get feedback, and work together to  find solutions to some of the challenges you face.
```
Comment (illicitonion) [inline docs/fundamentals/week-2/session.md]: It looks like there's a double-space here - I've made a line suggestion which you can press a button to accept, and GitHub will apply my suggestion to your PR and remove the extra space :)

```suggestion
Talk through your fears, get feedback, and work together to find solutions to some of the challenges you face. 
```

**Review by illicitonion — APPROVED** [review]
This looks like a great exercise, thanks so much for making a pull request to the syllabus! 🙏 

I have one tiny suggestion, then let's merge this in! 🎉 🎉 🎉 

This repo auto-deploys, so once we press merge, it should be live in syllabus.codeyourfuture.io in about 5 minutes :)

File: docs/fundamentals/week-2/session.md [hunk 2]
```diff
@@ -145,7 +145,41 @@ Read ahead to Ship It to see where to paste your code.
...
+
+**20 minutes: **
+
+In pairs take turns to ask the questions below:  
+When it comes to showing your work in progress, what are your three biggest fears?
+When it comes to showing your work in progress, how will you overcome your three biggest fears? 
+When it comes to showing your work in progress, what’s the absolute worst thing that can happen? 
+When it comes to showing your work in progress, what is the best thing that can happen? 
+When it comes to the working world, how will you support your team mates?
+When it comes to the working world, what do you need from your team mates?
```
Comment (alismith-dev) [inline docs/fundamentals/week-2/session.md]: I would worry that the first question is a bit hard to answer without context.

I wonder if we should swap these around and frame it as "what do you need, what will you do to support your team mates needs" would help with that?

File: docs/fundamentals/week-2/session.md [hunk 3]
```diff
@@ -145,7 +145,41 @@ Read ahead to Ship It to see where to paste your code.
...
+
+After 20 Minutes, it’s time to swap. 
+
+If you’re feeling brave, share your challenges to the group and ask for feedback. 
+
+
+**10 minutes**
+
+Task for class: 
+Each person has to give one piece of advice to support their fellow team members.
```
Comment (alismith-dev) [inline docs/fundamentals/week-2/session.md]: How would this be structured? Group discussion? Pairs? Something else?

**Review by alismith-dev — APPROVED** [review]
Great job for your first PR! 🎉 I left a couple of questions, but we can address those later (or we can practice updating a PR!)

File: docs/fundamentals/week-2/session.md (hunk 2 above)
Comment (ElizabethlawalCYF) [reply docs/fundamentals/week-2/session.md]: That's a great suggestion! 

Let's make that change

Comment (alismith-dev) [reply docs/fundamentals/week-2/session.md]: Great! 🙂 Do you need a hand with doing that? (I'm afraid I'm unlikely to have much time this week to help but I'm sure you could ask one of the other syllabus team members)

Comment (ElizabethlawalCYF) [reply docs/fundamentals/week-2/session.md]: Thanks Ali! All done :)

File: docs/fundamentals/week-2/session.md (hunk 3 above)
Comment (ElizabethlawalCYF) [reply docs/fundamentals/week-2/session.md]: done 👍

**Review by alismith-dev — APPROVED** [review]
LGTM :shipit:

**Comment by 40thieves** [comment]
@all-contributors please add @ElizabethlawalCYF for content

**Pull request merged**